app/frontend/src/AppRoute.tsx

```diff
@@ -46,6 +47,11 @@ export default function AppRoute({
 
   const classes = useStyles();
 
+  const RouteContent = useMemo(
+    () => () => <ErrorBoundary>{children}</ErrorBoundary>,
+    [children]
+  );
```

Comment:
I don't think there's any benefit to memoizing this.

Reply:
I see, you switched to `component` instead of `render`. If you do `render={() => <RouteContent>{children}</RouteContent>`, you won't need to memoize.

Reply:
Ok I drop the useMemo. Not sure if I understand though if there is a difference between:
`render={() => <RouteContent />}`
and
`component={RouteContent}`